backend/src/models/contracts.js

```diff
@@ -14,27 +15,28 @@ const contractSubcontractorTable = `${dataBaseSchemas().data}.contract_subcontra
 
 // Get all.
 const findAll = () => {
-  return knex
-    .select(
-      "contract.contract_number",
-      "contract.co_version",
-      "contract.description",
-      "supplier.supplier_name",
-      "contract.start_date",
-      "contract.end_date",
-      "contract.total_expense_amount",
+  return knex(`${contractsTable} as c`)
+    .columns(
+      "p.project_number",
+      "p.project_name",
+      "c.contract_number",
+      "c.co_version",
+      "c.description",
+      { supplier: "supplier.supplier_name" },
+      { start_date: knex.raw(`TO_CHAR(c.start_date :: DATE, '${dateFormat}')`) },
```

Comment:
Does this work with table sorting? Seems like it would cause 01-01-2022 to end up before 02-01-1900 after sorting for example. I wonder if it's better to have dates returned from the API as actual JavaScript Date objects rather than strings and then have the frontend determine what format to display them in (https://mui.com/x/react-data-grid/column-definition/#column-types for example).

Reply:
Approving the PR. Just wanted to mention there may be other types that also have sorting issues that we should look into (money type for example does string sorting instead of numeric sorting).